webrtc.html

```diff
@@ -9287,7 +9287,15 @@ <h4 id="sctp-transport-connected">Connected procedure</h4>
                   <p>Let <var>channel</var> be the <code><a>RTCDataChannel</a></code> object.</p>
                 </li>
                 <li>
-                  <p>If the <var>channel</var>'s <a>[[\DataChannelId]]</a> slot is greater or
+                  <p>If <var>channel</var>'s <a>[[\DataChannelId]]</a> slot is
+                    <code>null</code>, initialize <a>[[\DataChannelId]]</a>
+                    to a value generated by the
+                    user agent, according to [[!RTCWEB-DATA-PROTOCOL]].
+                    If no available ID could be generated,
+                    <a>throw</a> an <code>OperationError</code> exception.
```

Comment:
You can't throw an exception here, we're in a queued task, we're not in a function

Reply:
We should fail like we do below?

Reply:
Done.